mayavi/tests/test_mlab_integration.py

```diff
@@ -352,16 +349,9 @@ class TestMlabPipeline(TestMlabNullEngine):
     """
 
     def setUp(self):
-        ver = tvtk.Version()
-        self.less_than_or_equal_to_vtk_5_10 = True
-        if ver.vtk_major_version >= 5 and ver.vtk_minor_version > 10:
-            self.less_than_or_equal_to_vtk_5_10 = False
-        if self.less_than_or_equal_to_vtk_5_10:
-            super(TestMlabPipeline, self).setUp()
```

Comment:
I'm not sure of this change - because `ver.vtk_major_version >= 5 and ver.vtk.minor_version > 10` will always return `False` for the supported vtk versions i.e. vtk >= 7 but the changes I made are based on my understanding of how the code is written i.e. `if vtk version < 5.10 ... else ...`